firefox-ios/Storage/DefaultSuggestedSites.swift

```diff
@@ -33,100 +33,102 @@ open class DefaultSuggestedSites {
             ),
-            SuggestedSiteData(
+            SuggestedSite(
                 url: "http://sina.cn",
-                faviconResource: .remoteURL(url: URL(string: "https://mjs.sinaimg.cn/wap/online/public/images/addToHome/sina_114x114_v1.png")!),
-                trackingId: 702,
-                title: "新浪"
+                title: "新浪",
+                faviconResource: .remoteURL(url: URL(string: "https://mjs.sinaimg.cn/wap/online/public/images/addToHome/sina_114x114_v1.png")!)
             ),
-            SuggestedSiteData(
+            SuggestedSite(
                 url: "http://info.3g.qq.com/g/s?aid=index&g_f=23946&g_ut=3",
-                faviconResource: .remoteURL(url: URL(string: "https://mat1.gtimg.com/qqcdn/qqindex2021/favicon.ico")!),
-                trackingId: 703,
-                title: "腾讯"
+                title: "腾讯",
+                faviconResource: .remoteURL(url: URL(string: "https://mat1.gtimg.com/qqcdn/qqindex2021/favicon.ico")!)
             ),
-            SuggestedSiteData(
+            SuggestedSite(
                 url: "http://m.taobao.com",
-                faviconResource: .remoteURL(url: URL(string: "https://gw.alicdn.com/tps/i2/TB1nmqyFFXXXXcQbFXXE5jB3XXX-114-114.png")!),
-                trackingId: 704,
-                title: "淘宝"
+                title: "淘宝",
+                faviconResource: .remoteURL(url: URL(string: "https://gw.alicdn.com/tps/i2/TB1nmqyFFXXXXcQbFXXE5jB3XXX-114-114.png")!)
             ),
-            SuggestedSiteData(
+            SuggestedSite(
                 url: """
                 https://union-click.jd.com/jdc?e=618%7Cpc%7C&p=JF8BAKgJK1olXDYDZBoCUBV\
                 IMzZNXhpXVhgcCEEGXVRFXTMWFQtAM1hXWFttFkhAaihBfRN1XE5ZMipYVQ1uYwxAa1cZb\
                 QIHUV9bCUkQAF8LGFoRXgcAXVttOEsSMyRmGmsXXAcAXFdaAEwVM28PH10TVAMHVVpbDE8\
                 nBG8BKydLFl5fCQ5eCUsSM184GGsSXQ8WUiwcWl8RcV84G1slXTZdEAMAOEkWAmsBK2s
                 """,
+                title: "京东",
                 // FIXME We need a higher quality favicon link
-                faviconResource: .remoteURL(url: URL(string: "https://corporate.jd.com/favicon.ico")!),
-                trackingId: 705,
-                title: "京东"
+                faviconResource: .remoteURL(url: URL(string: "https://corporate.jd.com/favicon.ico")!)
             )
         ]
     ]
+
+    public static func defaultSites() -> [Site] {
+        let locale = Locale.current
+        let defaultSites = sites[locale.identifier] ?? sites["default"]
+        return defaultSites?.map { data in
+            if let domainMap = DefaultSuggestedSites.urlMap[data.url], let localizedURL = domainMap[locale.identifier] {
+                return SuggestedSite(url: localizedURL,
+                                     title: data.title,
+                                     faviconResource: data.faviconResource)
+            }
+            return data
+        } ?? []
+    }
```

Comment:
Also, this is much cleaner than the Site Cursor from before, nice work!! 🔥

Reply:
Thank you @ih-codes